### ISSUE TITLE
False Positive S3854: conditional execution

### ISSUE THREAD

**Issue by vilchik-elena** [SonarSource/SonarTS#176]
RSPEC-3854 (rules is implemented by tslint)
```typescript
constructor(
 item: ICommandAction,
 alt: ICommandAction,
 options: IMenuActionOptions,
 @ICommandService commandService: ICommandService
) {
       typeof item.title === 'string' ? super(item.id, item.title, commandService) : super(item.id, item.title.value, commandService);
...
}

```

**Comment by vilchik-elena** [comment]
it's pretty a corner case, so closing the issue
